torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -244,6 +246,19 @@ void StepMarker(const std::string& device_str,
   Device device = GetDeviceOrCurrent(device_str);
   XLATensor::SyncLiveTensorsGraph(&device, devices, wait);
   XLATensor::MarkStep(device);
+  bool debug_mode = xla::sys_util::GetEnvBool("PT_XLA_DEBUG", false);
+  if (TF_PREDICT_FALSE(debug_mode)) {
```

Comment:
ah this `TF_PREDICT_FALSE` is nice, didn't know this before.

Reply:
TF has everything 😄